yamllint/rules/key_ordering.py

```diff
@@ -63,15 +67,59 @@
 
     - haïr: true
       hais: true
+
+#. With ``key-ordering: {locale: "en_US.UTF-8"}``
+
+   the following code snippet would **PASS**:
+   ::
+
+    - key 1: v
+      key 2: val
+      key 3: value
+    - {a: 1, b: 2, c: 3}
+    - t-shirt: 1
+      T-shirt: 2
+      t-shirts: 3
+      T-shirts: 4
+    - hair: true
+      haïr: true
+      hais: true
+      haïssable: true
+
+   the following code snippet would **FAIL**:
+   ::
+
+    - T-shirt: 1
+      t-shirt: 2
+      T-shirts: 3
+      t-shirts: 4
+
+   the following code snippet would **FAIL**:
+   ::
+
+    - t-shirt: 1
+      t-shirts: 2
+      T-shirt: 3
+      T-shirts: 4
+
+   the following code snippet would **FAIL**:
+   ::
+
+    - hais: true
+      haïr: true
```

Comment:
I'm not sure we need all these, it could burden the documentation for users. What about just keeping the accented examples?

Reply:
I think the case example should be in there somehow. That's what initially motivated my changes - and the result is different from before now in two ways, because capital letters are now sorted **after** their lower counterparts. They were sorted **before** all lower letters before. So not only case-sensitive, but also "the other way around".

I think we can just keep the first sample (reduced to the t- and h- lines) that passes. Everyone can guess the failing lines themselves with more extended examples above.

What do you think?